nixos/modules/system/boot/find-libs.nix

```diff
@@ -0,0 +1,46 @@
+{ writeShellScriptBin, patchelf }:
```

Comment:
can we split out the refactoring of moving these two scripts into different files? I'd love to pick them into master already (as long as they are a no-op). This reduces the amount of changes this PR introduces and we can rule out some issues during the transition.

Reply:
find-libs gets removed in a later commit. The rust program handles that work now.